### PR TITLE
implement react site

### PR DESCRIPTION
**Pull request by antonfilka**

### CODE REVIEW

File: src/components/Cards/Cards.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import debounce from "lodash.debounce";
+import Card from "./Card/Card";
+import NoCardsPlug from "./NoCardsPlug/NoCardsPlug";
+
+
+const getFilteredCards = (query, cards) => {
+    return (
+        cards.filter((card) => {
+            return (
+                card.title.toLowerCase().includes(query) ||
```
Comment (antonfilka) [inline src/components/Cards/Cards.jsx:12]: delete return ()

File: src/components/Cards/Cards.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                card.text.toLowerCase().includes(query)
+            )
+        }).map((card) => {
+            return (
```
Comment (antonfilka) [inline src/components/Cards/Cards.jsx:16]: delete return ()

File: src/components/Cards/Cards.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+
+const getFilteredCards = (query, cards) => {
+    return (
+        cards.filter((card) => {
+            return (
+                card.title.toLowerCase().includes(query) ||
+                card.text.toLowerCase().includes(query)
+            )
+        }).map((card) => {
```
Comment (antonfilka) [inline src/components/Cards/Cards.jsx:15]: rewrite filterCards

File: src/components/Cards/Cards.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    const updateQuery = (e) => setQuery(e.target.value);
+    const debouncedOnChange = debounce(updateQuery, 300);
+
+    return(
+        <div className={classes.main}>
+            <div className={classes.search}>
+                <input type="search" placeholder="Enter what to find..." onChange={debouncedOnChange}/>
+            </div>
+            <div className={classes.cards}>
+                {filteredCards}
```
Comment (antonfilka) [inline src/components/Cards/Cards.jsx:46]: .map

File: src/components/Cards/Cards.jsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        }).map((card) => {
+            return (
+                <Card key={card.id.toString()} img={card.img} title={card.title} text={card.text}/>
+            )
+        })
+    )
+}
+
+
+const Cards = (props) => {
```
Comment (antonfilka) [inline src/components/Cards/Cards.jsx:33]: no props

File: src/components/Header/Tabs/DropdownTab/DropdownTab.jsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import classes from "./DropdownTab.module.css";
+
+
+const DropdownTab = (props) => {
```
Comment (antonfilka) [inline src/components/Header/Tabs/DropdownTab/DropdownTab.jsx:5]: no props
no return

File: src/components/SideBar/SideBar.jsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
+import React, {useState} from "react";
+import classes from "./SideBar.module.css";
+
+
+const SideBar = (props) => {
+
+    const overlayOpen = {height: '100%'};
```
Comment (antonfilka) [inline src/components/SideBar/SideBar.jsx:7]: clsx lib

File: src/components/SideBar/SideBar.jsx [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    return(
+        <div className={classes.overlay} style={props.overlayIsActive ? overlayOpen : overlayClose}>
+            <img src="https://flaticons.net/icon.php?slug_category=mobile-application&slug_icon=close" alt="img" onClick={() => props.setOverlayIsActive(false)} className={classes.closebtn} />
+            <div className={classes.overlayContent}>
+                {
+                    props.tabs.map((tab, index) => {
+                        if (tab.isDropdown) {
+                            return (
```
Comment (antonfilka) [inline src/components/SideBar/SideBar.jsx:26]: ? : 
separate components

File: src/components/SideBar/SideBar.jsx [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+                                <div>
+                                    <div className={classes.isDropdown}>
+                                        <a href={tab.link}>{tab.tabTitle}</a>
+                                        <img src="https://icon-library.com/images/white-down-arrow-icon/white-down-arrow-icon-7.jpg" onClick={ () => onclickHandler(index, props.tabs)} alt="img" style={tab.isActive ? rotateArrow : rotateArrow}/>
```
Comment (antonfilka) [inline src/components/SideBar/SideBar.jsx:30]: use State 
(id, active)

**Review by kuzahaha — APPROVED** [review]

**Pull request merged**